germany_fidorbank/README.rst

```diff
@@ -40,38 +36,40 @@ example:
 
   pip3 install --user ofxstatement
   git clone https://github.com/MirkoDziadzka/ofxstatement-germany
-  cd ofxstatement-germany/germany_1822direkt
+  cd ofxstatement-germany/germany_fidorbank
   pip3 install --user .
 
-Check the Python documentation on instructions for you operating system and
-setup. Remember you must use Python 3.
+Check the Python documentation on instructions for your operating system and
+setup. Remember, you must use Python 3.
 
 
 Setup
 =====
 
-Check if plugin is installed:
+Check if the plugin is installed:
 
 .. code:: bash
 
   ofxstatement list-plugins
 
 Expected output::
 
-  germany_1822direkt
+  germany_fidorbank
 
 Edit config. The *account* is the ID used by your accounting program to
 associate the transactions with a certain account. Probably you want to use
 your bank account number (Kontonummer) i.e. the last 10 digits of your IBAN.
+(Note: YNAB does not seem to care about this field, since you import directly
+to the account screen anyway.)
```

Comment:
At least in classic YNAB, you can import a file by starting YNAB with this file as an argument. This is what my import process is doing. It then uses the account data to find out into which of the YNAB accounts it should import this file.

Reply:
I'm using this with the web version of YNAB, so less relevant. It might still do this from the All Accounts screen, but I haven't tried.